tests/unit/stream_alert_cli/test_config.py

```diff
@@ -14,67 +14,73 @@
 limitations under the License.
 """
 # pylint: disable=protected-access
-from contextlib import nested
 import json
 
 from mock import Mock, patch
 from nose.tools import assert_equal, assert_true
+from pyfakefs import fake_filesystem_unittest
 
 from stream_alert_cli.config import CLIConfig
-from tests.unit.helpers.base import basic_streamalert_config, mock_open
+from tests.unit.helpers.base import basic_streamalert_config
 
 
 class TestCLIConfig(object):
     """Test class for CLIConfig"""
+    # pylint: disable=no-self-use
```

Comment:
We typically follow the format of one `Test...` class per file, so please add this pylint message to the top of the file alongside the `pylint: disable=protected-access` one that is currently there